pages/documentation/simulation/powerflow/openlf.md

```diff
@@ -10,6 +10,46 @@ PowSyBl OpenLoadFlow is an open-source power flow implementation in Java provide
 * TOC
 {:toc}
 
+## Grid modelling
+
+OpenLoadFlow computes power flows from IIDM grid model in bus/view topology. From the view, a very simple network, composed of only buses and branches is created. In the graph vision, we rely on a `Pi` model for branches (lines, transformers, dangling lines, etc.):
+
+- `R` and `X` are respectively the real part (resistance) and the imaginary part (reactance) of the complex impedance ;  
+- `G1` and `G2` are the real parts (conductance) on respectively side 1 and side 2 of the branch ;
+- `B1` and `B2` are the imaginary parts (susceptance) on respectively side 1 and side 2 of the branch ;
+- `A1` is the angle shifting on side 1, before the series impedance. For classical branches, the default value is zero ;
+- `R1` is the ratio of voltages between side 2 and side 1, before the series impedance. For classical branches, the default value is `1`.
+
+As the `Pi` model is created from IIDM grid modelling that locates its ratio and phase tap changers in side 1, `A2` and `R2` are always equal to zero and `1`. In case of a branch with voltage or phase control, the `Pi` model becomes an array. See below our model:
+
+![Pi model](img/pi-model.svg){: width="50%" .center-image}
```

Comment:
@Hadrien-Godard Can you please try to start from this model to add phase and ratio tap changer ?

Reply:
Done in cf112685569d166d678807236176641dd11e2b0b